### PR TITLE
[DX-1437] US-EN licence --> license

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: scripts/uk_us_mapping.csv.full-list [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Behaviour,Behavior
+Centre,Center
+Colour,Color
+Customisation,Customization
+Customise,Customize
+Defence,Defense
+Enrol,Enroll
+Favour,Favor
+Grey,Gray
++ Licence,License
```
Comment (dcs3spp) [inline scripts/uk_us_mapping.csv.full-list]: Is the + symbol used to enable that replacement?

File: scripts/convert_uk_to_us.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```
Comment (dcs3spp) [inline scripts/convert_uk_to_us.sh]:
```suggestion

#===============================================================================
#
#       FILE: convert_uk_to_us.sh
# 
#       USAGE: ./run-menu-gen.sh <csv_file> <directory>
# 
#       DESCRIPTION: This script reads a CSV file containing pairs of UK and US
#                words, then recursively searches through the specified directory
#                to replace all instances of the UK words with their US counterparts
#                in the files found.
# 
#       OPTIONS: 
#.               <csv_file> - Path to the CSV file containing UK and US word pairs.
#                <directory> - Path to the directory where replacements should be made.
#
#       REQUIREMENTS: The CSV file should be formatted with each line containing 
#                two words separated by a comma, where the first word is the UK 
#                word and the second is the US word. The script also requires 
#                write permissions for the files in the target directory.
#  
#       NOTES: The script uses `find` to search for files and `sed` to perform 
#                the replacements. It handles basic error checking and ensures 
#                that the specified CSV file and directory exist before proceeding.
#===============================================================================
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks Yaara, PR LGTM. I checked through the word replacements in PR and seems ok.

Have made minor suggestion for adding doc comment for usage in the bash script and updated file permissions for script to be executable.

Asked question about + character in csv file for the line containing license

File: scripts/uk_us_mapping.csv.full-list (hunk 1 above)
Comment (letzya) [reply scripts/uk_us_mapping.csv.full-list]: to mark it as done

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**